src/Uno.UITest.Puppeteer/SeleniumDriverManager.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Edge;
+
+namespace Uno.UITest.Selenium
+{
+	public class SeleniumDriverManager
+	{
+		public static class Chrome
+		{
+			// Chrome driver selection: http://chromedriver.chromium.org/downloads/version-selection
+
+			public static ChromeDriver FromChromePath(string chromePath, ChromeOptions options)
+				=> new ChromeDriver(
+					new SeleniumDriverManager("chromedriver").GetOrDownloadLatestDriverForBin(
+						chromePath ?? $@"{Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86)}\Google\Chrome\Application\chrome.exe",
+						GetDriverLatestVersion,
+						GetDriverUri).FullName,
+					options);
```

Comment:
Given that #53 was merged, you probably need to update this too.

Here's the code in question that is being used right now:
```
// Chrome might be installed in C:\Program Files\Google...
// If file doesn't exist, check there.
if(!File.Exists(chromePath))
{
	// Using environment variable here since EnvironMent.SpecialFolder.ProgramFiles resolves to the X86
	// variant depending on the executable architecture. The path variable always evaluates to the correct path though.
	chromePath = $@"{Environment.GetEnvironmentVariable("ProgramW6432")}\Google\Chrome\Application\chrome.exe";
}
```

Reply:
Handled